packages/react-core/src/components/Dropdown/DropdownMenu.tsx

```diff
@@ -180,7 +177,6 @@ export class DropdownMenu extends React.Component<DropdownMenuProps> {
       isGrouped,
       setMenuComponentRef,
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Is this comment still necessary?

Reply:
does this also apply to the comment on line 130 as well @nicolethoen ?

Reply:
i does not